### ISSUE TITLE
ordered and unordered lists recognized where there are none

### ISSUE THREAD

**Issue by g3o2** [DavidAnson/markdownlint#34]
The following text triggers linting rules MD 29 and 30/32.

```markdown
This example text triggers linting rules number 29 and
32. If no sentence follows the previous one, linting rules 29 and 30 are triggered.
```

A possible solution would be to search first for dot (`.`) in the line preceding the supposed ordered list. Only when `.` is found should rules 29, 30/32 be triggered.

Case of falsely recognised unordered list:

```markdown
This example text triggers linting rule number 32
- and sometimes rule number 4.
```

**Comment by DavidAnson** [comment]
After reading the spec, I learned that un/ordered lists *do* interrupt a paragraph - with the exception of numbers other than "1.". The specification explains this well: http://spec.commonmark.org/0.27/#lists

Based on that, you can verify that the second of the examples above is (correctly) parsed as an unordered list: https://markdown-it.github.io/

I'll have a fix for the first of the examples above tomorrow. Thanks for reporting this!
